formOptions.js

```diff
@@ -0,0 +1,28 @@
+const ebgOptions = [
```

Comment:
rename file to, constants.js and name variable as erasebgOptions

Reply:
changes done